nrich-notification/README.md

```diff
@@ -51,26 +51,26 @@ convenient way of returning a notification with a response.
 
 ## Usage
 
-nrich-notification is used nrich as a dependency of nrich-webmvc module. It is used in a `RestConstrollerAdvice`
-and on exceptions returns notifications to the client, so a common usage would be to use it in a `ControllerAdvice` to have unified error notification handling, a simple implementation of former
-advice would look something like this:
+`nrich-notification` is used nrich as a dependency of `nrich-webmvc` module. It is used in a `RestConstrollerAdvice` and on exceptions returns notifications to the client,
+so a common usage would be to use it in a `ControllerAdvice` to have unified error notification handling, a simple implementation of former advice would look something like this:
```

Comment:
"a simple implementation of former advice would look something like this:" should be a new sentence.